File: auction_service/src/main/java/com/example/auction/model/dao/AddressEntity.java
package com.example.auction.model.dao;

import lombok.Data;
import lombok.Getter;

import javax.persistence.*;

@Data
@Entity
@Table (name = "address")
public class AddressEntity {

    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private Integer Id;

    private String county;

    private String city;

    private String street;

    private int numberOfHouse;

    private int numberOfFlat;

    private String postCode;

}

File: auction_service frontend/auction/src/app/AddressEntity.ts
export class AddressEntity {

  county: String;
  city: String;
  street: String;
  numberOfHouse: number;
  numberOfFlat: number;
  postCode: String;

  constructor(county: String, city: String, street: String, numberOfHouse: number, numberOfFlat: number, postCode: String) {

    this.county = county;
    this.city = city;
    this.street = street;
    this.numberOfHouse = numberOfHouse;
    this.numberOfFlat = numberOfFlat;
    this.postCode = postCode;
  }
}